servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/SourceToFuture.java

```diff
@@ -91,7 +94,12 @@ public final boolean isDone() {
     public final T get() throws InterruptedException, ExecutionException {
         final Object value = this.value;
         if (value == null) {
-            latch.await();
+            lastGetTimestamp = System.currentTimeMillis();
```

Comment:
Any thoughts on `System.currentTimeMillis()` vs `System.nanoTime()`? I don't think for debugging precision matters, but I'm hesitant with possibility of `System.currentTimeMillis()` to return t2 < t1 for two consecutive calls.

Reply:
We only using it as a timestamp so I don't think we need monotonic guarantees.